src/config/constants.ts

```diff
@@ -18,7 +18,7 @@ export const TOKEN_INFO_URL = 'https://api.mycryptoapi.com/tokens';
 // The URL for MYC api.
 export const MYC_API = 'https://mycryptoapi.com/api/v1';
 
-export const HISTORY_API = 'https://history.mycryptoapi.com/history';
+export const HISTORY_API = 'https://history.mycryptodev.com/history';
```

Comment:
Is this intentional?

Reply:
leaving on dev till this pr review is pretty much done, then deploying production infra to make sure i dont have to redeploy prod infra

Reply:
prod infra deployed and this has been changed